lesson_04_homework.rb

```diff
@@ -1,50 +1,99 @@
-# Code Reading
+#Code Reading
 
 def fizzbuzz(num)
-  # Example comment line
+  # defines method with 1 parameter
 
   case
+    #modulo(%) finds the remainder after division
+    #if you divide the number by 15, 3 or 5 and there is no remainder the respective string below will be returned
     when num % 15 == 0 then "FizzBuzz"
+      #when number is multiple of 15 return "FizzBuzz"
     when num % 3  == 0 then "Fizz"
+      #when number is multiple of 3 return "Fizz"
     when num % 5  == 0 then "Buzz"
+      #when number is multiple of 5 return "Buzz"
     else num
+      #if the number is not a multiple of 15, 3, or 5 return the number
   end
 end
 
 def fizz_buzz_to(limit)
+  #defines method with 1 parameter
   1.upto(limit).each do |num|
+    #the upto iterator runs the method from 1 to whatever number is entered as limit
     puts fizzbuzz(num)
+    #calls fizzbuzz method above
   end
 end
+#the below runs the fizzbuzz method from 1 to 100
+fizz_buzz_to(100)
 
-# Fix Broken Code
 
-def area_of_triangle(b, h)
-  puts b x height / 2
+
+#Fix Broken Code
+
+def area_of_triangle(base, height)
+  base * height / 2
 end
 
-base = 7
-height = 6
+puts area_of_triangle(7, 6)
+
 
-puts area_of_triangle(height, height)
 
 # Coding
 
 class Waitlist
   # add your code here
+  attr_accessor :list
```

Comment:
@dnori817 good use of `attr_accessor` - now that you have this, you can refer to `@list` as `list`